### ISSUE TITLE
Generate random plausible comments

### ISSUE THREAD

**Issue by neilellis** [tessalt/echo-chamber-js#6]
Generate random plausible comments at random intervals, so the user has something to react against.

Some suggestions are.

"I like your article, it is very clever, do you want to buy some pills http://mypillsarecheap.com"

"I cannot honestly agree with this."

"I totally agree with the OP"

"I think you have a spelling mistake in the third sentence"

"Seriously, this is how the Nazis started"

...

**Comment by virtadpt** [comment]
+1

**Comment by kinduff** [comment]
Looks like we could fork out `faker` module and extend it with this kind of comments.

**Comment by nicolewhite** [comment]
:+1: for using [faker](https://github.com/marak/Faker.js/)

**Comment by MartijnR** [comment]
:+1:

**Comment by tra38** [comment]
:+1: I want to help in generating such comments though, especially with multi-paragraph rants that only really tells whether the commenter agreed with or disagreed with the poster.

**Comment by tessalt** [comment]
so here's what I was thinking: 

- author can populate with some default comments of their own 
- make it possibly integrate with hubot plugins or similar???? 
- I had not heard of faker, looks cool

**Comment by kinduff** [comment]
Maybe we can generate lists and then construct sentences with it, something like:
```
[agreeing or disagreeing] this article, but [random excuse]. [random comment]
[random comment]. [random excuse] but [agreeing or disagreeing part].
[agreeing or disagreeing] but [random excuse]. [random comment].
```
That's what I got right now and it could be implemented with `faker` pretty easy.

**Comment by virtadpt** [comment]
:+1: for Faker.

What about something like https://github.com/substack/node-markov trained on Youtube comments or something to generate fake responses?

**Comment by thejduce** [comment]
I'm not a coder, but I would thoroughly enjoy authoring fake comments or sections of comments if it came to that. Just tell me what format and I will unleash my best snark. :)

**Comment by tthomass** [comment]
Eliza might be a fun and unexpected source of fake comments

**Comment by stig** [comment]
:+1:

**Comment by jsweet** [comment]
@tthomass Is it because of your mother that you say Eliza might be a fun and unexpected source of fake comments?

Now I'm all paranoid that many if not most sites I post to are secretly using echo chamber. Including this one. Especially this one, really.

**Comment by leejinhuey** [comment]
+1

**Comment by drewbrew** [comment]
@jsweet How does your mother's reaction make you feel when you say that Eliza might be a fun and unexpected source of fake comments?

**Comment by twosticks** [comment]
Was going to suggest only one auto-generated comment.

USER: this is echo-chamber, isn't it?
EC: Yes.

Other than that it should not reply...

**Comment by manbearwiz** [comment]
+1 for a trollbot that always takes the opposite opinion of the OP. People would waste so much time!

**Comment by AsmodeusNOIR** [comment]
+1

**Comment by ShawnMilo** [comment]
10/10, would laugh.

**Comment by gkrnours** [comment]
Would it be possible to use the current user past comments and some markov magic to produce fake comments ?

**Comment by stevenmichaelthomas** [comment]
:+1:

**Comment by Arood** [comment]
@gkrnours nah, markov chains needs quite a lot of data to sound even remotely trustworthy. I'd go for making a library of YouTube-comments or similar that gets added randomly.

**Comment by securingsincity** [comment]
:+1:

**Comment by keithmo** [comment]
You could integrate it with an Eliza clone (such as elizabot-js) to actively engage the user in a super-annoying manner.

**Comment by jacksonofalltrades** [comment]
+100

**Comment by tobsn** [comment]
don't take it too far - just make a simple username + comment list and have it store on first load. then people can just customize whatever they want to "post" ahead of other people so it looks more relevant to the content. that alone would be golden...

**Comment by SimonAronsson** [comment]
I agree with KeithMo: Using Eliza for this would be awesome.

**Comment by tra38** [comment]
I found https://gist.github.com/Otto42/5467402 while browsing the GenerativeText Google Group. It is a series of spam comment templates that could be repurposed for this Echo-Chamber. All you have to do is just translate it to javascript.

**Comment by blunatic** [comment]
Usernames can be generated from the username field via http://www.fakenamegenerator.com/

**Comment by kinduff** [comment]
@blunatic: That's why `faker` was suggested.

**Comment by likexoo** [comment]
+10086
